src/main/java/com/powsybl/openloadflow/TargetVoltageCompatibilityChecker.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow;
+
+import com.google.common.base.Stopwatch;
+import com.powsybl.openloadflow.adm.AdmittanceEquationSystem;
+import com.powsybl.openloadflow.adm.AdmittanceMatrix;
+import com.powsybl.openloadflow.equations.VariableSet;
+import com.powsybl.openloadflow.network.LfBranch;
+import com.powsybl.openloadflow.network.LfBus;
+import com.powsybl.openloadflow.network.LfNetwork;
+import org.apache.commons.lang3.mutable.MutableInt;
+import org.apache.commons.lang3.tuple.Pair;
+import org.jgrapht.Graph;
+import org.jgrapht.graph.Pseudograph;
+import org.jgrapht.traverse.BreadthFirstIterator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
+ */
+public class TargetVoltageCompatibilityChecker {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TargetVoltageCompatibilityChecker.class);
+
+    private final LfNetwork network;
+
```

Comment:
Would it make sense to extend the API so that it can also be used with a Network (not an LfNetwork) and returns the IDM view of IIDM incompatibleControlled Buses - for example pairs of list of controllers  that are incomaptible) ?

Reply:
Yes, done. 
```java
Network network = ...;
LoadFlowParameters parameters = new LoadFlowParameters();
var elementIds = RemoteVoltageTargetChecker.findElementsToDiscardFromVoltageControl(network, parameters);
System.out.println(elementIds);
```

Reply:
Thanks !
Closing that one. (but I added an ohter commet on the method suggesting to add an explanation)